packages/typography/src/Typography.tsx

```diff
@@ -136,4 +137,145 @@ const Overline: ExtendableBox<{
   return <Box className={cx(sharedStyles, overline, className)} {...rest} />;
 };
 
-export { H1, H2, Subtitle, Body, InlineCode, Disclaimer, Overline };
+const OpenInNewTab = (props: JSX.IntrinsicElements['svg']) => {
```

Comment:
Curious if it'd be better to make a standalone `A.tsx` file or `AnchorElement`?

Reply:
Kk!